tests/workflows/test_uber_lyft.py

```diff
@@ -0,0 +1,67 @@
+import uuid
+
+import coiled
+import dask.dataframe as dd
+import pytest
+from dask.distributed import Client
+
+
+@pytest.fixture(scope="module")
+def uber_lyft_cluster(
+    dask_env_variables,
+    cluster_kwargs,
+    github_cluster_tags,
+):
+    with coiled.Cluster(
+        f"uber-lyft-{uuid.uuid4().hex[:8]}",
+        environ=dask_env_variables,
+        tags=github_cluster_tags,
+        **cluster_kwargs["uber_lyft_cluster"],
+    ) as cluster:
+        yield cluster
+
+
+@pytest.fixture
+def uber_lyft_client(
+    uber_lyft_cluster,
+    cluster_kwargs,
+    upload_cluster_dump,
+    benchmark_all,
+):
+    n_workers = cluster_kwargs["uber_lyft_cluster"]["n_workers"]
+    with Client(uber_lyft_cluster) as client:
+        uber_lyft_cluster.scale(n_workers)
+        client.wait_for_workers(n_workers)
+        client.restart()
+        with upload_cluster_dump(client), benchmark_all(client):
+            yield client
+
+
+@pytest.fixture
+def ddf(uber_lyft_client, read_parquet_with_pyarrow):
+    """NYC taxi Uber/Lyft dataset"""
+    return dd.read_parquet(
+        "s3://coiled-datasets/uber-lyft-tlc/",
+        engine="pyarrow",
+        storage_options={"anon": True},
+    )
+
+
+def test_mean_tips(ddf):
+    """How many passengers tip"""
+    (ddf.tips != 0).mean().compute()
+
+
+def test_max_pay(ddf):
+    """Max driver pay"""
+    ddf.driver_pay.max().compute()
+
+
+def test_max_distance(ddf):
+    """Ride count per rideshare company"""
+    ddf.groupby("hvfhs_license_num").count().compute()
+
+
+def test_longer_trips(ddf):
+    """How many passengers ride over 5 miles"""
+    (ddf.trip_miles > 5).mean().compute()
```

Comment:
Let's move these all into a single test. We'll eventually move the workflow in this file into its own notebook example that we want to be representative of what a common Dask DataFrame user might do as part of an exploratory analysis. I think we want a single test here that captures what that would look like. My current thinking is something relatively straightforward along the lines of (clearly it's just pseudocode below):

```python
# Load data
ddf = dd.read_parquet(...)

# Compute interesting quantities to gain insight
tips_fraction = (ddf.tips != 0).mean()
x = ...
y = ...
results = dask.compute(tips_fraction, x, y, ...)
```